native-schema-registry/python/PyGsrSerDe/KafkaSerDe.py

```diff
@@ -0,0 +1,49 @@
+from .PyGsrSerDe import *
+from .AvroSerDe import *
+
+class GlueSchemaRegistryKafkaSerializer:
+    def __init__(self):  # data format design needs discussion
+        self.data_format = "AVRO"
+        self.gsr_serializer = GlueSchemaRegistrySerializer
+        serializers = {
+            "AVRO": AvroSerializer(),
+            "JSON": "JSONSerializer()",  # placeholders
+            "PROTOBUF": "ProtobufSerializer()",
+        }
+        self.data_format_serializer = serializers.get(self.data_format)
+
+    def serialize(self, data: dict, schema: dict, topic: str):
```

Comment:
Is it the case that Python Kafka client producer expects a method with these parameters as callback? Can you link the doc for this?

Reply:
Updated per our conversation with configuration file, have separate methods for confluent as well.